### PR TITLE
Take Quiz Timer and Styles

### PR DESCRIPTION
**Pull request by gbmyt**
User can now set their own quiz timer

Styling tweaks for the begin screen

Fixed Pagination component (no numbers, user can now just click through to each Question)

### CODE REVIEW

File: Dockerfile
Comment (thanhgly) [inline Dockerfile:1]: Is pm2 still available? I noticed you removed it from server debs.

**Review by thanhgly — COMMENTED** [review]
Pulled and reviewed.
Things I noticed:
- When clicking the back button from the first question the page crashes.
- Whenever I go to the next question a new timer is created and overlapped with the current one.
-  I noticed you removed pm2 from the server debs but pm2 is still being used in Dockerfile.
- Some compile warnings that prevented the CI to pass.
- Everything else is working as expected. 

If everything is as planned then it's ok to proceed. However, I think the compile warnings should be fixed before merging to prevent the main branch from having CI issues.

**Comment by gbmyt** [comment]
@thanhgly Thanks for reviewing! Some notes:

- Fixing question takes down the Take Quiz Screen on 'back' button click, and buggy timer issues in the next PR.
- Removed pm2 from the server Dockerfile

Re: Compile Warnings-- I'm breaking the CI tests by importing axios in my custom hook on the client side. I believe this can be resolved by lifting the `@babel/preset-env` import from the server package.json to root + tweaking the install script to install root deps (in addition to the client/server deps which it's currently doing). 

This meant tweaking the CI config file to run the new script instead of default `npm install`. Can you vet this please, since you're the one who wrote up the CI? Don't want to break anything. 

Thank you!

**Comment by gbmyt** [comment]
Rolling back the CI tweak and npm install imports since Benny fixed separately in a different PR

**Review by thanhgly — APPROVED** [review]

**Pull request merged**